README.md

```diff
@@ -6,44 +6,69 @@
 [![Coverage Status](https://coveralls.io/repos/github/PlasmaPy/PlasmaPy/badge.svg?branch=master)](https://coveralls.io/github/PlasmaPy/PlasmaPy?branch=master)
 [![Matrix](https://matrix.to/img/matrix-badge.svg)](https://riot.im/app/#/room/#plasmapy:matrix.org)
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/PlasmaPy/Lobby)
+[![astropy](http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat)](http://www.astropy.org/)
 
-An open source community developed Python 3.6+ package for plasma physics in
-the early stages of development. PlasmaPy intends to be for plasmas what
-[Astropy](https://github.com/astropy/astropy) is for astronomy - a
-collection of functionality commonly used and shared between plasma physicists
-and researchers globally, running within and leveraging the open source
-scientific Python ecosystem. The goals of this project are more thoroughly described in our
-[vision statement](http://docs.plasmapy.org/en/stable/about/vision_statement.html)
-and [an earlier conference poster](http://doi.org/10.5281/zenodo.163752).
+PlasmaPy is an open source community developed Python 3.6+ package for
+plasma physics in the early stages of development.  PlasmaPy intends to
+be for plasmas what [Astropy](https://github.com/astropy/astropy) is for
+astronomy - a collection of functionality commonly used and shared
+between plasma physicists and researchers globally, running within and
+leveraging the open source scientific Python ecosystem.  The goals of
+this project are more thoroughly described in our [vision
+statement](http://docs.plasmapy.org/en/stable/about/vision_statement.html)
+and [a recent reference](https://doi.org/10.5281/zenodo.1238132).
 We are in the process of writing [online
 documentation](http://plasmapy.readthedocs.io/en/latest/).
 
-We created a guide on
-[contributing to PlasmaPy](http://plasmapy.readthedocs.io/en/master/CONTRIBUTING.html) and have a
-[Code of Conduct](http://plasmapy.readthedocs.io/en/master/CODE_OF_CONDUCT.html).
+We created a guide on [contributing to
+PlasmaPy](http://plasmapy.readthedocs.io/en/master/CONTRIBUTING.html)
+and have a [Code of
+Conduct](http://plasmapy.readthedocs.io/en/master/CODE_OF_CONDUCT.html).
 New contributors are very welcome!
 
-If you have any questions, the quickest way to get a response is to ask on our
+If you have any questions, the quickest way to get a response is to ask
+on our
 [Matrix](https://riot.im/app/#/room/#plasmapy:matrix.org)/[Gitter](https://gitter.im/PlasmaPy/Lobby)
-channel.  You may also contact
-Nick Murphy at <namurphy@cfa.harvard.edu> and
-Drew Leonard at <andy.j.leonard@gmail.com>.
+channel.  For a slower response, you may also contact Nick Murphy at
```

Comment:
Figured it's probably best to be honest, given how slow I am at sending and responding to emails...  We still need to create some @plasmapy.org email addresses, though that feature costs a non-negligible amount per month for when we essentially have no budget...

Reply:
I think we're fine without those email addresses for now! Setting those up would likely be way more trouble than it's worth. 

Also, we could just add the mailing list/google group here. This would probably solve a few problems with chat being volatile in nature and possibly unfavorable for prolonged discussion.